File: app/src/main/java/id/wisata/Place.java
package id.wisata;

public class Place {

    private String Name;
    private int ImageUrl;
    private String Address;
    private Float Rating;

    public Place() {

    }

    public Place(String name, int imageUrl, String address, Float rating){
        Name = name;
        ImageUrl = imageUrl;
        Address = address;
        Rating = rating;
    }

    public String getName() {
        return Name;
    }

    public int getImageUrl() {
        return ImageUrl;
    }

    public String getAddress() {
        return Address;
    }

    public Float getRating() {
        return Rating;
    }

    public void setName(String name) {
        Name = name;
    }

    public void setImageUrl(int imageUrl) {
        ImageUrl = imageUrl;
    }

    public void setAddress(String address) {
        Address = address;
    }

    public void setRating(Float rating) {
        Rating = rating;
    }
}

File: app/src/main/java/id/wisata/Place.kt
package id.wisata

class Place {

    var name: String? = null
    var imageUrl: Int = 0
    var address: String? = null
    var rating: Float? = null

    constructor() {

    }

    constructor(name: String, imageUrl: Int, address: String, rating: Float?) {
        this.name = name
        this.imageUrl = imageUrl
        this.address = address
        this.rating = rating
    }
}
